sql/src/main/java/io/crate/execution/dml/upsert/ShardUpsertRequest.java

```diff
@@ -225,6 +213,8 @@ public void writeTo(StreamOutput out) throws IOException {
 
         sessionSettings.writeTo(out);
 
+        boolean allOn4_2 = out.getVersion().onOrAfter(Version.V_4_2_0);
```

Comment:
Maybe also inline this, I think there is only a single use.

Reply:
It is used for the serialization of all the items.

Reply:
Ah right, I missed that.